Formula/zorba.rb

```diff
@@ -24,10 +24,14 @@ class Zorba < Formula
 
   conflicts_with "xqilla", because: "both supply `xqc.h`"
 
-  def install
-    # icu4c 61.1 compatibility
-    ENV.append "CXXFLAGS", "-DU_USING_ICU_NAMESPACE=1"
+  # Fixes for missing headers and namespaces from open PR in GitHub repo linked via homepage
+  # PR ref: https://github.com/zorba-processor/zorba/pull/19
+  patch do
+    url "https://github.com/zorba-processor/zorba/commit/e2fddf7bd618dad9dc1e684a2c1ad61103b6e8d2.patch?full_index=1"
```

Comment:
This is 1st commit in PR that fixes icu4c namespace issue (previous work around was `U_USING_ICU_NAMESPACE=1`) and missing headers that break Linux build.